packages/falcon-sensor/overlay/questions.yaml

```diff
@@ -0,0 +1,98 @@
+questions:
+  - variable: node.image.repository
+    description: "URL of container image repository holding containerized Falcon sensor. Defaults to 'falcon-node-sensor'."
+    required: true
+    type: string
+    default: falcon-node-sensor
+    label: Container Image Repository
+    group: "Node Container Images"
+
+  - variable: node.image.tag
+    description: "Container registry image tag. Defaults to 'latest'."
+    required: true
+    type: string
+    default: "latest"
+    label: Container Image Tag
+    group: "Node Container Images"
+      
+  - variable: falcon.cid
```

Comment:
The questions are fine, but having CLI flags as labels look a bit out of place in the UI (The UI uses this to generate the text fields for each question). Could you please update them to have a brief description in the label, and then use the description for additional details that might be necessary like the CLI flag it represents? (See examples below)

Reply:
Thanks @PennyScissors! I believe that I have fixed these. Let me know if not.